react/features/base/media/middleware.js

```diff
@@ -45,7 +49,13 @@ MiddlewareRegistry.register(store => next => action => {
         const result = next(action);
         const { track } = action;
 
-        track.local && _syncTrackMutedState(store, track);
+        // Don't sync track mute state with the redux store for screenshare
+        // since video mute state represents local camera mute state only.
+        // Also, do not try to sync mute state for the presenter track since
```

Comment:
How is that possible? Since we run `next(action)` above the redux store should be up to date already. Can you please clarify?